frontend/amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -232,7 +233,15 @@ export class TableDetail extends React.Component<
 
     return descriptions.map((d) => (
       <EditableSection key={`prog_desc:${d.source}`} title={d.source} readOnly>
-        <EditableText maxLength={999999} value={d.text} editable={false} />
+        <div className="editable-text">
```

Comment:
I am not sure what are we doing here.
You modified <EditableText> to allow for the 'allowDangerousHtml' prop, why using ReactMarkdown here directly?